docs/04-refactor-state-management.md

```diff
@@ -32,7 +32,6 @@ Now that this type is registered in DI, we can `@inject` it into the Index page.
 @inject HttpClient HttpClient
 @inject OrderState OrderState
 @inject IUriHelper UriHelper
-@implements IDisposable
```

Comment:
Disposable is no longer needed here because we don't need to pass around change notifications.